### PR TITLE
Accesibility fixes

### PR DESCRIPTION
**Pull request by MarekBodingerBA**

### CODE REVIEW

**Review by radoslavzeman — APPROVED** [review]
Please just make sure that `useId` does not cause any errors :)

File: next/public/locales/sk/common.json [hunk 1]
```diff
@@ -207,8 +207,7 @@
     "noReviews": "Žiadne recenzie na zobrazenie."
   },
   "DocumentGroup": {
-    "download": "Stiahnuť",
-    "downloadFile": "Stiahnuť súbor"
```
Comment (radoslavzeman) [inline next/public/locales/sk/common.json:211]: You also need to update en version to pass the assert check

File: next/components/molecules/DocumentGroup.tsx [hunk 2]
```diff
@@ -30,14 +34,14 @@ const DocumentGroup = ({ documents }: DocumentGroupFragment) => {
                 <Button
                   href={file.data?.attributes?.url}
                   variant="tertiary"
-                  // TODO add file size and format
-                  aria-label={t('downloadFile')}
+                  aria-labelledby={getAriaLabelId(id, index)}
```
[the remark this answers is not in the document]
Comment (MarekBodingerBA) [reply next/components/molecules/DocumentGroup.tsx:37]: aria-label can be used in cases where text that could label the element is not visible. If there is visible text that labels an element, use [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) instead.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

[the remark this answers is not in the document]
Comment (MarekBodingerBA) [reply next/components/molecules/DocumentGroup.tsx:37]: I set the JIRA to TODO.

[the remark this answers is not in the document]
Comment (radoslavzeman) [reply next/components/molecules/DocumentGroup.tsx:37]: Okay, if that's what docs say, no problem about `aria-labeledby` :)

**Review by radoslavzeman — APPROVED** [review]

**Pull request merged**